### PR TITLE
Adding fs tc 11331

### PR DESCRIPTION
**Pull request by julpark-rh**
Steps:
    1. set the mds conf journal values
    2. verify the mds conf journal values
    
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-WYC5V7

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/clients/mds_journal_value_conf_verification.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Resetting the client config values to default")
+        for default in conf_default:
+            client_conf = default[0]
+            value = default[1]
+            client1.exec_command(sudo=True, cmd=f"{conf_set} {client_conf} {value}")
+        log.info("Successfully reset all the client config values to default")
```
Comment (hkadam134) [inline tests/cephfs/clients/mds_journal_value_conf_verification.py:93]: Clean up of mounts, mounting dirs are missing

**Comment by neha-gangadhar** [comment]
@julpark-rh Please add the test to reef folder suite file as well.

File: tests/cephfs/clients/mds_journal_value_conf_verification.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        ]
+        for conf in conf_default:
+            output = client1.exec_command(sudo=True, cmd=f"{conf_get} {conf[0]}")
+            conf[1] = output[0].strip()
+        conf_target = [
+            ("journaler_write_head_interval", "16"),
+            ("journaler_prefetch_periods", "12"),
+            ("journaler_prezero_periods", "6"),
+        ]
+        with parallel() as p:
```
Comment (neha-gangadhar) [inline tests/cephfs/clients/mds_journal_value_conf_verification.py:65]: I think the IOs must be run post setting custom values to journal parameters and check no crashes or errors in the cluster health.

Comment (hkadam134) [reply tests/cephfs/clients/mds_journal_value_conf_verification.py:65]: @neha-gangadhar  - We are validating the on-the-fly config changes while IOs are on. 
we can extend the IO duration to run for a longer duration or run the IOs again once the settings are changed.

Comment (neha-gangadhar) [reply tests/cephfs/clients/mds_journal_value_conf_verification.py:65]: Noted, can we validate crash and health status to ensure no issues with conf changes?

File: tests/cephfs/clients/mds_journal_value_conf_verification.py (hunk 1 above)
Comment (julpark-rh) [reply tests/cephfs/clients/mds_journal_value_conf_verification.py:98]: added

File: tests/cephfs/clients/mds_journal_value_conf_verification.py (hunk 2 above)
Comment (julpark-rh) [reply tests/cephfs/clients/mds_journal_value_conf_verification.py:65]: added health status after the conf chagnes

**Review by neha-gangadhar — APPROVED** [review]
lgtm

**Pull request merged**